mantidproject/mantid
# Update LoadWAND to use h5py and add grouping option

Replace use of `LoadEventNexus` and `Integration` with `h5py`, same as `LoadWANDSCD`. This will also enable quick grouping of pixels. With `Grouping='None'` the loaded data should be exactly the same as before.

Grouping pixels to `4x4` shows a speed improvement in `WANDPowderReduction` from ~2min to ~10sec with no significant difference in reduced data.


**To test:**

Try the [`LoadWAND`](http://docs.mantidproject.org/nightly/algorithms/LoadWAND-v1.html#usage) and [`WANDPowderReduction`](http://docs.mantidproject.org/nightly/algorithms/WANDPowderReduction-v1.html#usage) usage examples to see that data is loaded faster. Then try with setting `Grouping='2x2'` and `Grouping='4x4'` and compare.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
